src/main/java/ladder/Points.java

```diff
@@ -0,0 +1,63 @@
+package ladder;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Points {
+    private static final Random random = new Random();
+
+    private List<Point> points = new ArrayList<>();
+
+    public Points(int count) {
+        for (int i = 0; i < count; i++) {
+            addPoint();
+        }
+    }
+
+    public Points(List<Point> points) {
+        this.points = points;
+    }
+
+    public List<Boolean> connections() {
+        return points.stream()
+                .map(Point::connected)
+                .collect(Collectors.toList());
+    }
+
+    private static boolean randomBoolean() {
+        return random.nextBoolean();
+    }
+
+    private void addPoint() {
+        points.add(new Point(connectable()));
+    }
+
+    private boolean connectable() {
+        if (points.isEmpty()) return randomBoolean();
```

Comment:
```suggestion
        if (points.isEmpty()) {
            return randomBoolean();
        }
```
1줄짜리 if문이라도 중괄호는 넣어주세요. 가독성에 큰 도움이 됩니다.